torch_geometric/utils/scatter_reduce/composite/scatter.py

```diff
@@ -0,0 +1,49 @@
+import os
+from typing import Optional, Tuple
+
+import torch
+from torch import Tensor
+
+from .logsumexp import scatter_logsumexp
+from .softmax import scatter_log_softmax, scatter_softmax
+from .std import scatter_std
+
+
+def scatter_composite(src: Tensor, index: Tensor, dim: int = -1,
```

Comment:
Once PyTorch support for broadcasting is in place, we can slowly get users used to using PyTorch `scatter` (instead of the wrapper). Atter that procedure, only `scatter_composite` would be left in PyG API.